### DISCUSSION TITLE
Documentation around -ocr option

### DISCUSSION THREAD

**Discussion by gaffinet** [sepinf-inc/IPED#194] [Q&A] [answered]
Hi

I have an indexed case and now I want to ORC a subset of the files.

The command line tool gives following hint:
```
-ocr
      only run OCR on a specific category or bookmark (can be used multiple times)
```
But I have no clue how to construct the entire command.

When I use -ocr in combination with -d and --append I get the error message that the case (in the -o folder) already contains the given evidence.

When I use -ocr without -d, I get the error message that I need to specify an evidence.

May you please complete the ORC chapter in documentation of the wiki

Thank you

**Comment by lfcnassif** [comment]
Unfortunately it is not possible to run OCR or any other module after the case is processed. For now you must configure all options before creating the case. This is The major limitation of the tool and will start to be addressed next month in #24 

About documentation, yeah, it is very incomplete, will try to enhance it in the next weeks.

**Comment by gaffinet** [comment]
Thanks for the reply

But I cannot understand the help message: How can I only ocr a selected set bookmarked files, when I need to process the evidence to bookmark the files?

Or, do I need to export the case somehow (with the bookmarks) an then reprocess this export again with the -ocr option?

**Comment by lfcnassif** [comment, marked as answer]
You can restrict ocr, when first processing, to some categories:
Iped.exe -d image.dd -o case_folder -ocr "PDF Files"

Or when creating the portable indexed report from the cmd line, you can restrict ocr to some bookmarks:
Iped.exe -d case_folder/indexador/marcadores.iped -o report_folder -ocr bookmark1 -ocr bookmark2

marcadores.iped is the file that controls your bookmarks.

enableOCR must be true in IpedCondig.txt config in both cases.

This could be exposed by report gui dialog, but running ocr just when reporting is just useful in specific scenarios when other user are going to analyze the data extracted by the examiner.
